pkg/instance/instance_test.go

```diff
@@ -20,6 +20,7 @@ func TestExecprogCmd(t *testing.T) {
 	flags := flag.NewFlagSet("", flag.ContinueOnError)
 	flagOS := flags.String("os", runtime.GOOS, "target os")
 	flagArch := flags.String("arch", "", "target arch")
+	flagType := flags.String("type", "", "target VM type")
```

Comment:
This test mocks an outdated syz-execprog binary, so you should not have added this flag.

Reply:
Got it, thanks. I've made this an optional flag and updated the comment in the test; does the new comment look correct? And is there a process for transitioning optional flags to required ones eventually?